torch/serialization.py

```diff
@@ -103,6 +103,29 @@ def storage_to_tensor_type(storage):
 def save(obj, f, pickle_module=pickle, pickle_protocol=DEFAULT_PROTOCOL):
     """Saves an object to a disk file.
 
+    There are two main approaches for serializing and restoring a model.
```

Comment:
I'd say that just "model" is better. NNs are special cases of models in general 😉